test/correctness/simd_op_check.cpp

```diff
@@ -1889,6 +1889,9 @@ struct Test {
         check("v*:*.w = vtmpy(v*:*.h, r*.b)", hvx_width/2, i32(in_i16(x - 1))*2 + i32(in_i16(x)) + i32(in_i16(x + 1)));
         check("v*:*.w = vtmpy(v*:*.h, r*.b)", hvx_width/2, i32(in_i16(x - 1)) + i32(in_i16(x)) + i32(in_i16(x + 1)));
 
+        check("v*:*.w = vrmpy(v*:*.ub, r*.b, #*)", hvx_width/1, 2*i32(in_u8(x - 1)) + 3*i32(in_u8(x)) + 5*i32(in_u8(x + 1)) + 7*i32(in_u8(x + 2)));
+        check("v*:*.w = vrmpy(v*:*.ub, r*.b, #*)", hvx_width/1, 9*i32(in_u8(x - 1)) + 8*i32(in_u8(x)) + 7*i32(in_u8(x + 1)));
```

Comment:
What is the # syntax? Is this required to be an immediate value? If so, I think there should be some asserts in the code that generates vrmpy to make sure the scalars are constants.